GEOSens_GridComp/GEOS_EnsGridComp.F90

```diff
@@ -1604,6 +1604,24 @@ subroutine SetServices(gc, rc)
                                            RC=STATUS  )
   VERIFY_(STATUS)
 
+  call MAPL_AddExportSpec(GC                         ,&
+    LONG_NAME          = 'CN_autotrophic_respiration' ,&
+    UNITS              = 'kg m-2 s-1'                ,&
+    SHORT_NAME         = 'CNAR'                      ,&
+    DIMS               = MAPL_DimsTileOnly           ,&
+    VLOCATION          = MAPL_VLocationNone          ,&
+                                           RC=STATUS  )
+  VERIFY_(STATUS)
+
+  call MAPL_AddExportSpec(GC                         ,&
+    LONG_NAME          = 'CN_heterotrophic_respiration' ,&
```

Comment:
I suggest we call this "CN_total_heterotrophic_respiration".  Would also need to be changed in https://github.com/GEOS-ESM/GEOSgcm_GridComp/pull/1129

Reply:
Long name has been updated in the GEOSldas_GridComp and the GEOSgcm_GridComp repositories.